3rdparty/carotene/src/resize.cpp

```diff
@@ -758,7 +758,7 @@ inline void resizeAreaRounding(const Size2D &ssize, const Size2D &dsize,
     }
     else if (channels == 3)
     {
-        if ((wr == 2.0f) && (wr == 2.0f))
```

Comment:
According to `isResizeAreaSupported` that is called before hand to check HAL kernel applicability the condition should be `if ((wr == 2.0f) && (hr == 2.0f))`.